lib/data/external/GCP/GcpUtils.js

```diff
@@ -112,6 +113,78 @@ function createMpuList(params, level, size) {
     return retList;
 }
 
+function processTagSet(tagSet = []) {
+    if (tagSet.length > 10) {
+        return errors.BadRequest
+            .customizeDescription('Object tags cannot be greater than 10');
+    }
+    let error = undefined;
+    const tagAsMeta = {};
+    const taggingDict = {};
+    tagSet.every(tag => {
```

Comment:
I would suggest using a for loop and returning an error immediately if any case is true.

Reply:
Will modify to use a for loop and returning on error